### ISSUE TITLE
Unit test discovery throws errors, documentation should tell users how tests should be run

### ISSUE THREAD

**Issue by clarkkarenl** [dpkp/kafka-python#742]
I wanted to get started with the kafka-python client, so I decided to check out the unit tests.
I did this:
`cd kafka-python/test`
`python -m unittest discover`

and got this error (along with many, many others):

> $ python -m unittest discover
> # EEEEEEEEEEEEEEEEEEEEE
> ## ERROR: test_client (unittest.loader.ModuleImportFailure)
> 
> ImportError: Failed to import test module: test_client
> Traceback (most recent call last):
>   File "/usr/local/Cellar/python/2.7.11/Frameworks/Python.framework/Versions/2.7/lib/python2.7/unittest/loader.py", line 254, in _find_tests
>     module = self._get_module_from_name(name)
>   File "/usr/local/Cellar/python/2.7.11/Frameworks/Python.framework/Versions/2.7/lib/python2.7/unittest/loader.py", line 232, in _get_module_from_name
>     __import__(name)
>   File "/Users/karen/code/kafka-python/test/test_client.py", line 5, in <module>
>     from . import unittest
> ValueError: Attempted relative import in non-package

Is the unittest suite not intended to be executed by `discover`?

**Comment by dpkp** [comment]
That is a lot of E's!!!

Generally we run tests via tox:

`KAFKA_VERSION=9.0.0.1 tox -e 27`

Which may require running build_integration.sh to download server binaries

**Comment by dpkp** [comment]
Also note that we use pytest not unittest for a lot of test modules

**Comment by clarkkarenl** [comment]
Thanks @dpkp! I'd like to update the project wiki to mention this. Any objections? (Or should I do that via git / pull request?)

**Comment by dpkp** [comment]
By all means! Any and all improvements welcome

**Comment by clarkkarenl** [comment]
Here's the patch file, containing the edits. I wrote it using the WYSIWYG wiki editor but am happy to adjust the source if formatting is off at all. Thanks!
[kafka-python-issue742.patch.gz](https://github.com/dpkp/kafka-python/files/346786/kafka-python-issue742.patch.gz)

**Comment by clarkkarenl** [comment]
Alright, now that I fixed user.name in my git config...trying again. :)

Attached is a patch that addresses this issue in the quickest way possible:  the Home page of the wiki.
I also prettied up the page with some formatting to make things easier to read. Feel free to reject any of that you wish.
[kafka-python-issue742.patch.gz](https://github.com/dpkp/kafka-python/files/346800/kafka-python-issue742.patch.gz)

Thanks!
